test/test_suite

```diff
@@ -18,6 +18,15 @@ function simple_examples {
 
 run_with_flags simple_examples ""
 
+function simple_stdlib_tests {
+	for file in test/stdlib/*.dbl
+	do
+		simple_test $file
+	done
+}
+
+run_with_flags simple_stdlib_tests ""
```

Comment:
I agree, we should have some tests for stdlib. Maybe more conventional unit testing would be nice in the future. But now, this should be enough.